### DISCUSSION TITLE
Singleplayer spectating not working anymore

### DISCUSSION THREAD

**Discussion by minasminas** [ppy/osu#17266] [Q&A] [answered]
The singleplayer spectate doesn't seem to work anymore. It did work a few weeks ago (don't know which update broke it). Multiplayer spectate is working good as usual by the way.

While testing this issue I also made sure that the user is really playing the map and isn't in the menus or pausing.

Here is a short video of the issue:

https://user-images.githubusercontent.com/81645139/158482669-6692fe95-93f8-4b8b-a445-d02811147468.mp4

Logs:
[performance.log](https://github.com/ppy/osu/files/8257473/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8257475/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8257476/updater.log)
[database.log](https://github.com/ppy/osu/files/8257477/database.log)
[network.log](https://github.com/ppy/osu/files/8257478/network.log)

**Comment by peppy** [comment]
It does look to work, but you have to skip forward manually to the point the user is currently at. Seems like the seek logic broke?

**Reply by frenzibyte** [reply]
Indeed, have found the regressing commit.

**Comment by frenzibyte** [comment, marked as answer]
Escalated to #17267
